### ISSUE TITLE
CVE-2023-44487 vulnerability found in esbuild (golang version 1.20)

### ISSUE THREAD

**Issue by ApurvaHegde** [angular/angular-cli#27716]
### Command

other

### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

Our company uses the Black Duck Binary Analysis tool to scan vulnerabilities which shows that there is a high vulnerability for golang version 1.20.
The path to the vulnerability is shown as .../app/node_modules/@esbuild/linux-x64/bin/esbuild and .../package/bin/esbuild.

This vulnerability is attached to CVE-2023-44487 which is a known exploit.

### Minimal Reproduction

The code is run through the Black Duck Binary Analysis tool.

### Your Environment

```text
Angular CLI: 15.2.11
Node: 16.14.2
Package Manager: npm 8.5.0
OS: darwin arm64

Angular: 15.2.10
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.1502.11
@angular-devkit/build-angular      15.2.11
@angular-devkit/core               15.2.11
@angular-devkit/schematics         15.2.11
@angular/cdk                       15.2.9
@angular/cli                       15.2.11
@angular/material                  15.2.9
@angular/material-moment-adapter   15.2.9
@schematics/angular                9.1.15
rxjs                               7.8.1
typescript                         4.9.5
```

### Anything else relevant?

My present version of esbuild as shown by the command npm ls esbuild is "0.17.8". I have tried to override the esbuild version used by @angular-devkit/build-angular using this code in package.json.

"overrides": {
    "@angular-devkit/build-angular": {
      "esbuild": "0.21.4"
    }
  } 

I have used esbuild version 0.17 to 0.21 in this command and tried to run the build but this did not fix the issue. 

Is there version of esbuild used by @angular-devkit/build-angular which solves this issue?

**Comment by JeanMeche** [comment]
This was fixed by [evanw/esbuild/commit/cd91337d87e4f53e388a5a66d24e791957165b24](https://github.com/evanw/esbuild/commit/cd91337d87e4f53e388a5a66d24e791957165b24)

So any version >= v0.19.5 should be fine.

**Comment by alan-agius4** [comment]
Closing as version 15 is no longer supported. Please see: https://angular.dev/reference/releases#actively-supported-versions
